webapp/ordering/admin_views.py

```diff
@@ -198,9 +198,17 @@ def post(self, request, *args, **kwargs):
 
     def corrections(self):
         order_round = OrderRound.objects.get(pk=self.kwargs.get('pk'))
-        return OrderProductCorrection.objects.filter(
+        corrections = OrderProductCorrection.objects.filter(
             order_product__order__order_round=order_round).order_by(
-            "order_product__order__user")
+            "order_product__product__supplier","order_product__order__user")
+        correction_per_supplier = {}
+        for correction in corrections:
+            supplier = correction.order_product.product.supplier.name 
+            if supplier in correction_per_supplier:
```

Comment:
Tip: voor dit pattern kun je in Python ook de `defaultdict` gebruiken

Reply:
Dank je Rik, 
Mijn ervaring met Python en Django is beperkt, dus dit soort tips zijn van harte welkom.
Ik heb alles zonder problemen lokaal aan de praat gekregen. Het zit logisch in elkaa.
Deze change heb ik lokaal getest met verschillende leveranciers, producten en bestellingen in een order.